server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -762,6 +772,25 @@ public String serviceName(ServiceRequestContext ctx) {
         }
     }
 
+    private static Function<? super HttpService, CorsService> configCorsDecorator(List<String> allowedOrigins) {
+        if (allowedOrigins == null || allowedOrigins.isEmpty()) {
+            return null;
+        }
+
+        CorsServiceBuilder corsServiceBuilder;
+        if (allowedOrigins.contains("*")) {
+            corsServiceBuilder = CorsService.builderForAnyOrigin();
+        } else {
+            corsServiceBuilder = CorsService.builder(allowedOrigins);
+        }
+        return corsServiceBuilder
+                       .allowRequestMethods(HttpMethod.GET, HttpMethod.POST, HttpMethod.OPTIONS)
```

Comment:
Central Dogma REST API also uses `DELETE` and `PATCH`. `HttpMethod.knownMethod()` for simplicity.